gnomad/resources/grch37/reference_data.py

```diff
@@ -10,6 +10,92 @@
     import_sites_vcf,
 )
 
+
+def _import_gencode(gtf_path: str, **kwargs) -> hl.Table:
+    """
+    Import GENCODE annotations GTF file as a Hail Table.
+
+    :param gtf_path: Path to GENCODE GTF file.
+    :return: Table with GENCODE annotation information.
+    """
+    ht = hl.experimental.import_gtf(
+        gtf_path,
+        **kwargs,
+    )
+    # Only get gene and transcript stable IDs (without version numbers if they
+    # exist), early versions of GENCODE have no version numbers but later ones do.
+    ht = ht.annotate(
+        gene_id=ht.gene_id.split("\\.")[0],
+        transcript_id=ht.transcript_id.split("\\.")[0],
+    )
+    return ht
+
+
+def _extract_gencode_cds(gencode_path: str) -> hl.Table:
```

Comment:
I don't think this should be in resources, instead add it to filtering like we did for clinvar path variants. 

https://github.com/broadinstitute/gnomad_methods/blob/08b1a53cf6b7ba9837bd238c451ad66c97c382ec/gnomad/utils/filtering.py#L316